app/client/src/pages/logout.tsx

```diff
@@ -32,14 +32,15 @@ export default function Logout() {
                 onCompleted() {
                     resetEnv();
                     setUser(null);
+                    router.push('/login');
                 },
             });
         }
-    }, [runMutation, isClient, resetEnv, setUser]);
+    }, [runMutation, isClient, resetEnv, setUser, router]);
 
     React.useEffect(() => {
         let handle: NodeJS.Timeout | undefined;
-        if (!user) {
+        if (!user || user == null) {
```

Comment:
Curious as to why you needed the user == null? !user should cover that. Also you should use === not ==

Reply:
I used `user == null` because I thought `null` wasn't included in `!user` but I can change it and test it. I think I got an error because for some reason the user was returning null when logging out. And yes good point, I'll change `==` to `===`.